packages/package.go

```diff
@@ -387,6 +388,20 @@ func (p *Package) HasKibanaVersion(version *semver.Version) bool {
 	return p.Conditions.Kibana.constraint.Check(version)
 }
 
+func (p *Package) HasCapabilities(capabilities []string) bool {
+	if p.Conditions == nil || p.Conditions.Elastic == nil || p.Conditions.Elastic.Capabilities == nil || capabilities == nil {
+		return true
+	}
+
+	// TODO if there are several capabilities in the query, should all of them match with the ones defined in the package ?
+	for _, c := range capabilities {
+		if !util.StringsContains(p.Conditions.Elastic.Capabilities, c) {
```

Comment:
I think it should be the other way around. For each capability in the package, there should be a matching one in the declared capabilities.
```suggestion
	for _, requiredCapability := range p.Conditions.Elastic.Capabilities {
		if !util.StringsContains(capabilities, requiredCapabilities)
```

Reply:
So all the capabilities defined in the package must be present in the query in order to be added to the response ? @jsoriano

Reply:
Yes. Clients (aka Kibana) will indicate all the capabilities they support. We should only return packages whose required capabilities are satisfied in this list.

Reply:
Updated logic to do it like this.
Example in the description has also been updated taken into account this logic.

----

For the latter scenario, let's consider the following capabilities for a given package:
```yaml
conditions:
  elastic.capabilities:
    - observability
    - uptime
```
here it is shown if the package could be added to the response or not (let's consider that there are no other filters):
| Query parameter | Package can be added | 
| --- | --- |
| No capability in the query | yes |
| capabilities=observability | no |
| capabilities=observability,security | no |
| capabilities=observability,uptime | yes |
| capabilities=observability,uptime,security | yes |